src/EditorFeatures/VisualBasicTest/CodeActions/EncapsulateField/EncapsulateFieldTests.vb

```diff
@@ -129,14 +129,14 @@ End Class</File>.ConvertTestSourceTag()
 
             Dim expected = <File>
 Class C
-    Private _x As Integer
+    Private x1 As Integer
```

Comment:
📝 This all seems fine, but it would be good to have another test for a non-default naming